src/guide/extras/customFunctions.md

```diff
@@ -1,5 +1,33 @@
 # Custom Functions
-## Creating the custom function
```

Comment:
Revert.

Reply:
Why?

Reply:
It's just the title of a page, it shouldn't be too long.
Straightforward.

Reply:
Ok changing it.